Manager/src/main/java/io/raspberrywallet/manager/common/http/ApacheSecuredHttpClient.java

```diff
@@ -0,0 +1,20 @@
+package io.raspberrywallet.manager.common.http;
+
+import org.apache.commons.lang.NotImplementedException;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.fluent.Form;
+
+import java.io.IOException;
+
+public class ApacheSecuredHttpClient extends HttpClient {
```

Comment:
What is this class for ?

Reply:
It's for the future, right now auth server is using http and when https will be implemented, then I will change this to correct client

Reply:
Ok